invokeai/backend/stable_diffusion/diffusion/custom_atttention.py

```diff
@@ -30,23 +37,22 @@ def __init__(
                 for the i'th IP-Adapter.
         """
         super().__init__()
-        self._ip_adapter_weights = ip_adapter_weights
-
-    def _is_ip_adapter_enabled(self) -> bool:
-        return self._ip_adapter_weights is not None
+        self._ip_adapter_attention_weights = ip_adapter_attention_weights
 
     def __call__(
         self,
         attn: Attention,
-        hidden_states: torch.FloatTensor,
-        encoder_hidden_states: Optional[torch.FloatTensor] = None,
-        attention_mask: Optional[torch.FloatTensor] = None,
-        temb: Optional[torch.FloatTensor] = None,
-        # For regional prompting:
+        hidden_states: torch.Tensor,
+        encoder_hidden_states: Optional[torch.Tensor] = None,
+        attention_mask: Optional[torch.Tensor] = None,
+        temb: Optional[torch.Tensor] = None,
+        # For Regional Prompting:
         regional_prompt_data: Optional[RegionalPromptData] = None,
-        percent_through: Optional[torch.FloatTensor] = None,
+        percent_through: Optional[torch.Tensor] = None,
         # For IP-Adapter:
         regional_ip_data: Optional[RegionalIPData] = None,
+        *args,
+        **kwargs,
```

Comment:
Why add *args and **kwargs?

Reply:
So the override function matches the base and we don't get the incompatible override type checking errors in the editor.

Reply:
Good 'ol python types... I think the behaviour we'd really want here is to raise if anything was passed to `*args` or `**kwargs` since we aren't handling them